scripts/tests/data/studies.json

```diff
@@ -8,7 +8,7 @@
     }, 
     "description": "<p><span style=\"font-family:arial,helvetica,sans-serif\"><span style=\"font-size:24px\"><strong>Single nucleus RNA-seq of cell diversity in the adult mouse hippocampus.</strong></span></span></p>\r\n\r\n<p><span style
 =\"font-size:20px\"><span style=\"font-family:arial,helvetica,sans-serif\">Habib N, Li Y, Heidenreich M, Swiech L, Avraham-Davidi I, Trombetta J, Hession C, Zhang F,&nbsp;Regev A.&nbsp;<strong>Div-Seq: Single-nucleus RNA-Seq reveals dynam
-ics of rare adult newborn neurons.</strong>&nbsp;<cite>Science&nbsp;</cite>&nbsp;28 Jul 2016&nbsp;<a href=\"https://dx.doi.org/10.1126/science.aad7038\">DOI: 10.1126/science.aad7038</a><br />\r\nContact:&nbsp;naomi@broadinstitute.org</spa
+ics of rare adult newborn neurons.</strong>&nbsp;<cite>Science&nbsp;</cite>&nbsp;28 Jul 2016&nbsp;<a href=\"https://doi.org/10.1126/science.aad7038\">DOI: 10.1126/science.aad7038</a><br />\r\nContact:&nbsp;naomi@broadinstitute.org</spa
```

Comment:
@katrinleinweber, this URL is part of test data pulled via the [Single Cell Portal REST API](https://portals.broadinstitute.org/single_cell/api/swagger_docs/v1#!/Studies/studies_path). 
 The appropriate place to make the change would be [upstream in the study](https://portals.broadinstitute.org/single_cell/study/-single-nucleus-rna-seq-of-cell-diversity-in-the-adult-mouse-hippocampus-snuc-seq) description itself.  

As an SCP developer I can't make that upstream edit.  But feel free to suggest the edit to the author of the study (contact information is listed in linked study), then we can sync any updates here.

Reply:
I guess the authors of that study need to do it? I don't have an account with that portal.